src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -10,10 +10,21 @@ import type {
 
 const createReleaseServiceInstalls = async (
 	api: sbvrUtils.PinejsClient,
-	deviceIds: number[],
+	deviceFilterOrIds: number[] | FilterObj,
 	releaseFilter: Filter,
 ): Promise<void> => {
-	if (deviceIds.length === 0) {
+	const deviceIds = Array.isArray(deviceFilterOrIds)
```

Comment:
`deviceIds` is not used anywhere.

Reply:
Ugh, never mind -- it is used later on and in fact we do a `Promise.map` on the array which might contain 66k IDs? :/

Reply:
It's used in
```
	await Promise.all(
		deviceIds.map(async (deviceId) => {
```
below (typescript would complain if it was unused)

Reply:
It is still used near the end of the function, where we create the service installs missing for each device.